ci/Jenkinsfile

```diff
@@ -39,27 +25,18 @@ library identifier: "ci-pipeline@${env.ghprbActualCommit}",
 properties(
         [
                 buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '90', numToKeepStr: '')),
-                disableConcurrentBuilds(),
                 parameters(
                         [
-                                string(description: 'CI Message that triggered the pipeline', name: 'CI_MESSAGE'),
-                                string(defaultValue: 'f27', description: 'Fedora target branch', name: 'TARGET_BRANCH'),
-                                string(defaultValue: '', description: 'HTTP Server', name: 'HTTP_SERVER'),
-                                string(defaultValue: '', description: 'HTTP dir', name: 'HTTP_DIR'),
-                                string(defaultValue: 'ci-pipeline', description: 'Main project repo', name: 'PROJECT_REPO'),
-                                string(defaultValue: '', description: 'Main topic to publish on', name: 'MAIN_TOPIC'),
-                                string(defaultValue: '', description: 'Principal for authenticating with fedora build system', name: 'FEDORA_PRINCIPAL'),
                                 string(defaultValue: 'master', description: '', name: 'ghprbActualCommit'),
                                 string(defaultValue: 'CentOS-PaaS-SIG/ci-pipeline', description: '', name: 'ghprbGhRepository'),
-                                string(defaultValue: '', description: '', name: 'sha1'),
-                                string(defaultValue: '', description: '', name: 'ghprbPullId'),
-                                string(defaultValue: '', description: '', name: 'ghprbPullAuthorLogin'),
                                 string(defaultValue: 'stable', description: 'Tag for rpmbuild image', name: 'SLAVE_TAG'),
```

Comment:
nit - description for the container tags

Reply:
Good catch. Thought I had fixed that.